src/AWS.Deploy.Recipes/CdkTemplates/BlazorWasm/BlazorWasm.csproj

```diff
@@ -25,7 +25,7 @@
 
   <ItemGroup>
     <!-- CDK Construct Library dependencies -->
-    <PackageReference Include="Amazon.CDK.Lib" Version="2.13.0" />
+    <PackageReference Include="Amazon.CDK.Lib" Version="2.43.1" />
```

Comment:
Previously we had everything at 2.13.0, and that's the version we list in https://github.com/aws/aws-dotnet-deploy/blob/3bc852c68eaad581ab92a12cae8c6d93625e72ee/THIRD_PARTY_LICENSES#L20 too.

Should we update all the recipes to stay in sync? If not should we add a separate entry to the license file?

Reply:
Updated everything to version `2.43.1` including in the `THIRD_PARTY_LICENSES`.